packages/rum/src/domain/record/elementsScrollPositions.ts

```diff
@@ -24,3 +33,21 @@ export function createElementsScrollPositions() {
     },
   }
 }
+
+function findDocumentScrollingElement(childNodes: NodeList, scrollPositions: ScrollPositions) {
+  let scrollingElement: Element | null = null
+  forEach(childNodes, (node) => {
+    if (!scrollingElement && node.nodeType === node.ELEMENT_NODE) {
+      const element = node as Element
+      if (
+        Math.round(element.scrollTop) === scrollPositions.scrollTop &&
+        Math.round(element.scrollLeft) === scrollPositions.scrollLeft
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏If `scrollTop` and `scrollLeft` are 0, it would match any non-scrollable element.